frontend/public/components/dashboard/dashboards-page/cluster-dashboard/health-item.tsx

```diff
@@ -226,10 +227,14 @@ export const PrometheusHealthItem = withDashboardResources<PrometheusHealthItemP
   },
 );
 
-export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem }) => {
+export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem, namespace }) => {
   const { title, resources, healthHandler, popupComponent, popupTitle } = subsystem;
-  const resourcesResult = useK8sWatchResources(resources);
-
+  const resourcesWithNamespace = _.set(
+    _.cloneDeep(resources),
+    'imageManifestVuln.namespace',
+    namespace,
+  );
```

Comment:
can we have it as below?

```suggestion
const { imageManifestVuln : { kind : resourceKind} } = resources;
const resourcesWithNamespace = React.useMemo(() => {
  return {
        imageManifestVuln: {
            isList: true
            kind: resourceKind
            namespaced : true,
            namespace,
           }
      }
}, [resourceKind , namespace]);
```

Reply:
updated it to
```
const { kind : resourceKind} = resources?.imageManifestVuln ?? {};
const resourcesWithNamespace = React.useMemo(() => {
  return {
        imageManifestVuln: {
            isList: true
            kind: resourceKind
            namespaced : true,
            namespace,
           }
      }
}, [resourceKind , namespace]);
```
to account for undefined and null values

Reply:
updated it to 
`const resourcesWithNamespace: WatchK8sResources<ResourcesObject> = React.useMemo(() => {
    return {
      ...resources,
      ...(resources.imageManifestVuln && {
        imageManifestVuln: { ...resources.imageManifestVuln, namespace },
      }),
    };
  }, [resources, namespace]);`
as the previous implementation excluded the attributes other than `imageManifestVuln` and may cause errors in other components using this.